custom_components/tahoma/cover.py

```diff
@@ -113,6 +113,10 @@ def current_cover_position(self):
             CORE_TARGET_CLOSURE_STATE,
         )
 
+        # Position retrieved via TARGET_CLOSURE_STATE can be over 100 if unknown
+        if position > 100:
```

Comment:
What about `if position < 0 or position > 100`? In the comment you linked, we talked about a negative value.